Pp 5860 add smtp

danworth opened the pull request:
This enables simple deployment of Sentry to a cf space along with an smtp server to enable it to send emails. Some files have been copied from `paas-incubator/sentry` and modified for our purpose. I have tested it by running it in `sandbox`.

danworth committed:
PP-5860 Add an smtp server for sentry

To enable Sentry running on PaaS to send email this adds a simple SMTP
server to the space and enables connectivity to Sentry.

For more details please see `README.md`

danworth committed:
PP-5860 Make deploy script

`cli/deploy_sentry.sh` will deploy sentry and the required smtp server.
This can be used to set-up Sentry from scratch.

rjbaker reviewed (dismissed):
Looks good. Probably worth removing the existing files from this repo as they're not used.

danworth committed:
PP-5809 Removed legacy Sentry files

Removed some old files that are no longer used, from when
implementing Sentry was first attempted.

danworth commented:
> Looks good. Probably worth removing the existing files from this repo as they're not used.

Good idea, I've added a commit removing them 👍
